src/esp32/entry.rs

```diff
@@ -24,81 +24,76 @@ use esp_idf_svc::http::client::{Configuration as HttpConfiguration, EspHttpConne
 pub fn serve_web(
```

Comment:
All I did was remove the wrapping scope. Not sure why the diff looks all funny. 

On second thought, since this function never returns, keeping that wrapping scope might be good to drop lingering stuff on the stack (if any, need to look closer).

Reply:
updated commit shows minimal diff now